ludwig/datasets/mixins/kaggle.py

```diff
@@ -0,0 +1,40 @@
+import os
+from pathlib import Path
+from kaggle.api.kaggle_api_extended import KaggleApi
+
```

Comment:
Similar to this: https://github.com/horovod/horovod/blob/master/test/utils/common.py#L139

But we need to make one change, which is that we want to keep the original environment as well.

Reply:
```
@contextmanager
def update_env(**kwargs):
    override_env = {k: v for k, v in kwargs.items() if v is not None}
    old = os.environ
    try:
        os.environ = {os.environ, **override_env}
        yield
    finally:
        os.environ = old
```

Reply:
Added